admin/sql/create_tables.sql

```diff
@@ -1,11 +1,12 @@
 BEGIN;
 
 CREATE TABLE lowlevel (
-  id          SERIAL,
-  mbid        UUID NOT NULL,
-  build_sha1  TEXT NOT NULL,
-  lossless    BOOLEAN                  DEFAULT 'n',
-  submitted   TIMESTAMP WITH TIME ZONE DEFAULT NOW()
+  id              SERIAL,
+  mbid            UUID NOT NULL,
+  build_sha1      TEXT NOT NULL,
+  lossless        BOOLEAN                  DEFAULT 'n',
+  submitted       TIMESTAMP WITH TIME ZONE DEFAULT NOW(),
+  submission_type BOOLEAN NOT NULL         DEFAULT FALSE
```

Comment:
Also not sure if default is needed. This should be explicitly specified when adding a column.